cookbooks/mediawiki/templates/default/LocalSettings.php.erb

```diff
@@ -236,8 +236,11 @@ $wgShowIPinHeader = FALSE;
 # Job Runs by cron
 $wgJobRunRate = 0;
 
+# dissolves double redirects automatically
+$wgFixDoubleRedirects = TRUE;
+
 # Allow external images from a few sites
-$wgAllowExternalImagesFrom = array( 'http://tile.openstreetmap.org/', 'http://svenanders.openstreetmap.de/', 'http://josm.openstreetmap.de/', 'http://trac.openstreetmap.org/', 'http://rweait.dev.openstreetmap.org/' );
+$wgAllowExternalImagesFrom = array( 'http://tile.openstreetmap.org/', 'http://josm.openstreetmap.de/', 'http://trac.openstreetmap.org/', 'http://rweait.dev.openstreetmap.org/' );
```

Comment:
Why have you changed this line?

Reply:
> In addition, a reference to a non-functional web site is removed.

Reply:
Right, but that should be a separate edit, not tagged on to something unrelated...

Reply:
Should I open a separate pull request for that?

Reply:
Yes please - update this PR to remove and open a separate PR if you want to remove that,